geoportailv3/static/js/maincontroller.js

```diff
@@ -86,6 +90,16 @@ app.MainController.prototype.setMap_ = function() {
 
 
 /**
+ * @private
+ */
+app.MainController.prototype.updateSelectedLayers_ = function() {
+  var items_ = this['map'].getLayers().getArray().slice().reverse();
+  items_.pop();
+  this['selectedLayers'] = items_;
```

Comment:
This code deserves a comment (in the code) I think. The call to `pop` is obscure if you don't know that we want to exclude the background layers here.